.kitchen.yml

```diff
@@ -38,7 +38,7 @@ provisioner:
     - 139
   max_retries: 2
   remote_states:
-    name: git://github.com/saltstack/salt-jenkins.git
+    name: git://github.com/gtmanfred/salt-jenkins.git
```

Comment:
crap, this one should be saltstack

Reply:
https://github.com/saltstack/salt/pull/46991

Reply:
OK - just in the 2018.3 branch? Or a fix to 2017.7 is needed as well?